pyproject.toml

```diff
@@ -1,3 +1,90 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "cpg-flow"
+description = "CPG Flow API for Hail Batch"
+requires-python = ">=3.10"
+readme = "README.md"
+license = {"file"="LICENSE"}
+dynamic = ["version"]
+dependencies = [
+    # "cpg-utils>=5.0.11",
+    # "cyvcf2==0.30.18",
+    # "analysis-runner>=2.43.3",
+    # "hail==0.2.132",  # Pin Hail at CPG"s installed version
+    # "networkx>=2.8.3",
+    # "obonet>=0.3.1",  # for HPO parsing
+    # "grpcio-status>=1.48,<1.50",  # Avoid dependency resolution backtracking
+    # "onnx",
+    # "onnxruntime",
+    # "skl2onnx",
+    # "metamist>=6.9.0",
+    # "pandas",
+    # "peddy>=0.4.8",  # Avoid 0.4.7, which is incompatible
+    # "pyfaidx>=0.8.1.1",
+    # "fsspec",
+    # "slack_sdk",
+    # "elasticsearch==8.*",
+    # "coloredlogs",
+    # "bokeh",
+    # "numpy",
+    # "click",
+    # "tenacity",
+    # "toml",
```

Comment:
Seeing as the `requirements-dev.txt` file was deleted, should this bit be commented out here? Can see we have optional dependencies below so I guess it might be fine 🤔

Reply:
Again, my thought was to re-add dependencies in as we go. That way we don't have any unnecessary dependencies.